src/flb_metrics.c

```diff
@@ -320,6 +320,25 @@ static int attach_build_info(struct flb_config *ctx, struct cmt *cmt, uint64_t t
     return 0;
 }
 
+static int attach_hot_reload_info(struct flb_config *ctx, struct cmt *cmt, uint64_t ts,
+                                  char *hostname)
+{
+    double val;
+    struct cmt_gauge *g;
+
+    g = cmt_gauge_create(cmt, "fluentbit", "", "hot_reloaded_times",
+                         "Collect the count of hot reloaded times.",
+                         1, (char *[]) {"hostname"});
+    if (!g) {
+        return -1;
+    }
+
+    val = (double) ctx->hot_reloaded_count;
+
+    cmt_gauge_set(g, ts, val, 1, (char *[]) {hostname});
```

Comment:
I think this value should be initialized to zero if not reloads has been done (otherwise the metrics is missing until a reload happens)...or why is being set to 1 ?

Reply:
The 1 is for the count of label(s). Am I missing here?

Reply:
The default value of hot_reloaded_count is already set up by this line: https://github.com/fluent/fluent-bit/pull/7899/files#diff-fedd373b500bde2e636fa69cd909d04c9d1372654a2689cad142fe16c5663db5R277

Reply:
thanks, I got confused